web/templates/admin/dashboard.tmpl

```diff
@@ -85,5 +85,54 @@
     </div>
     {{end}}
   </div>
+  <div class="max-w-lg">
+    <h2 class="text-xl tracking-tight font-bold text-black mt-2 mb-2">{{i18n "PrivacyModesTitle"}}</h2>
+    <p class="mb-4">
+      {{ i18n "ExplanationPrivacyModes" }} 
+      <a class="text-purple-600 font-bold" href="https://ssb-ngi-pointer.github.io/rooms2/#privacy-modes">{{ i18n "RoomsSpecification" }}</a>.
+    </p>
+  <h3 class="text-gray-600 text-sm font-bold mb-2">{{ i18n "SetPrivacyModeTitle" }}</h3>
```

Comment:
```suggestion
  <h3 class="text-gray-400 text-sm font-bold mb-2">{{ i18n "SetPrivacyModeTitle" }}</h3>
```

To match the same style of the other dropdown in the member details page.

Reply:
i noticed this, and chose 600 because i thought it was the better choice ;)

but i will commit your suggestion

Reply:
We can actually change it (maybe 500?) in a separate PR, as long as they are consistent everywhere. My concern here was just to keep it consistent, I'm not adamant on 400.